File: CPP/Day12.cpp
/*

Day 12: Program to reverse the digits of given number.

*/ 


//Program
#include <iostream>
using namespace std;

int reverseNumber(int n) {
    int rev = 0;
    while (n > 0) {
        rev = rev * 10 + n % 10;
        n /= 10;
    }
    return rev;
}

int main() {
    int n;
    cout<<"Enter the number to be reversed: ";
    cin >> n;
    int rev = reverseNumber(n);
    cout << "The reversed number is: " << rev << endl;
    return 0;
}



/*

Input & Output:

Enter the number to be reversed: 12344
The reversed number is: 44321

*/

File: Python/Day12.py
"""

Day 12:	Program to reverse the digits of given number

"""

#Program
n = int(input("Enter the number: "))
m = n
rev = 0

while(n > 0):
	a = n % 10
	rev = rev * 10 + a
	n = n // 10

print(f"The number {m} after reversing:  ", rev)


"""

Input and Output: 
Enter the number: 44567
The number 44567 after reversing:   76544

"""
